tests/Responses/CakeResponseFactoryTest.php

```diff
@@ -22,14 +22,13 @@ public function testCreate()
         );
 
         $factory = new CakeResponseFactory();
+        /** @var \Cake\Http\Response|\Cake\Network\Response $response */
         $response = $factory->create($cache, 'kayaks.jpg');
 
-        $headers = $response->header();
-
         $this->assertInstanceOf('Cake\Network\Response', $response);
         $this->assertEquals('image/jpeg', $response->type());
-        $this->assertEquals('5175', $response->length());
-        $this->assertContains(gmdate('D, d M Y H:i', strtotime('+1 years')), $response->expires());
-        $this->assertEquals('max-age=31536000, public', $headers['Cache-Control']);
+        $this->assertEquals('5175', $response->getHeaderLine('Content-Length'));
```

Comment:
Would these call still work the same way in 3 and 4 (I don't know CakePHP very well)?

Reply:
My bad, it's tests

If that's OK for you, I'd like to fix the tests of this package before merging, so that we can ensure it won't break applications.